speech/unit_tests/test_client.py

```diff
@@ -258,9 +258,9 @@ def make_channel(*args):
             channel_args.append(args)
             return channel_obj
 
-        def speech_api(channel=None):
+        def speech_api(channel=None, **kwargs):
             return _MockGAPICSpeechAPI(response=_make_sync_response(),
-                                       channel=channel)
+                                       channel=channel, **kwargs)
```

Comment:
Because we accept something like 15 parameters, and more may be added in the future, and that should be able to be done without erroring out unrelated tests.